src/main/obsRecorder.ts

```diff
@@ -133,22 +133,51 @@ const setupScene = (monitorIndex: number) => {
 
   // Correct the monitorIndex. In config we start a 1 so it's easy for users. 
   const monitorIndexFromZero = monitorIndex - 1; 
-  console.info("monitorIndexFromZero:", monitorIndexFromZero);
+  console.info("[OBS] monitorIndexFromZero:", monitorIndexFromZero);
   const { physicalWidth, physicalHeight } = displayInfo(monitorIndexFromZero);
 
+  const obsSettings = osn.NodeObs.OBS_settings_getSettings('Video').data;
+
+  // Checks if val is +-1 compare
+  const isClose = (val: number, compare: number) => {
+    return Math.abs(compare - val) <= 1;
+  };
+
+  // Checks if resolution is close enough to monitor, and if so, sets the settingString.
+  const checkRes = (resolution: string) => {
```

Comment:
Prefer functions not to be nested in other functions. Also this comment isn't quite right here, this doesn't do any setting.